src/libs/OptionsListUtils.js

```diff
@@ -210,11 +210,7 @@ function hasReportDraftComment(report) {
  * @returns {Array<String>}
  */
 function getParticipants(report) {
-    const participants = [...lodashGet(report, ['participants'], [])];
-    if (ReportUtils.isPolicyExpenseChat(report) && !_.contains(participants, report.ownerEmail)) {
-        participants.push(report.ownerEmail);
-    }
-    return participants;
+    return [...lodashGet(report, ['participants'], [])];
```

Comment:
OK, now that we are not pushing anything into the participants the spread is not necessary? Probably this whole method can go and the one usage can be updated

https://github.com/Expensify/App/blob/47b4541391d2b9e70e003b3d58e401bc9ee5e974/src/libs/OptionsListUtils.js#L436